websites/Z/ZeroTwo/presence.ts

```diff
@@ -0,0 +1,26 @@
+const presence = new Presence({
+    clientId: "752067809214857298"
+  });
+  
+  presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+      largeImageKey: "logo"
+    };
+  
+    if ((await presence.getSetting("incognito")) === false) {
+      presenceData.details = document.getElementById("premid").innerText;
+      if ((await presence.getSetting("showTimestamp")) === true)
+        presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```

Comment:
init the timestamp outside of the updatedata event

Reply:
could you elaborate?

Reply:
> init the timestamp outside of the updatedata event

could you elaborate?

Reply:
> > init the timestamp outside of the updatedata event
> 
> could you elaborate?

If you define the timestamp in the `UpdateData` event the time counter will be always 00:00 because the timestamp will be always updating. Define it next to the `presence` on 4 line so that your counter works correctly.